sdks/java/io/elasticsearch-tests/elasticsearch-tests-7/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTest.java

```diff
@@ -83,6 +83,12 @@ public void setup() throws IOException {
 
   @Rule public TestPipeline pipeline = TestPipeline.create();
 
+  @Test
```

Comment:
IT feels wrong making a unit test in this file for the `SslConfiguration` class as all the other tests in this file are hitting an Elasticsearch cluster. I don't really have any need to do that here as we know that the ES client works with a valid SSL config. Really, I just want to test that SslConfiguration serializes properly and that pipelines still init as expected.